### PR TITLE
blog: Kronos - Run easily ABAP code for free in 5 minutes

### PR DESCRIPTION
**Pull request by iliyan-velichkov**
To review the blog execute `bundle exec jekyll serve` and open [http://localhost:4000/technology/2024/06/25/run-abap-for-free/](http://localhost:4000/technology/2024/06/25/run-abap-for-free/)

### CODE REVIEW

**Comment by iliyan-velichkov** [comment]
@delchev @ThuF the PR is for review

File: _posts/2024-06-25-run-abap-for-free.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+     <img src="{{ site.baseurl }}/images/2024-06-25-run-abap-for-free/steps-output-in-http-response.png">
+   
+This is how we ran our existing ABAP code using Kronos in a few quick steps.
+
+## Summary
+Let's summarize what you can do with Kronos
+- you can easily run ABAP code in Kronos
+- it costs you nothing since Kronos is open source project
+- licenses are not required
+- you can use any Database supported by codbex Atlas like H2, PostgreSQL, MySQL, Snowflake, HANA, etc.<br>
```
Comment (delchev) [inline _posts/2024-06-25-run-abap-for-free.md:375]: Atlas -> Kronos

**Review by delchev — CHANGES_REQUESTED** [review]
Awesome!

File: _posts/2024-06-25-run-abap-for-free.md (hunk 1 above)
Comment (iliyan-velichkov) [reply _posts/2024-06-25-run-abap-for-free.md]: done

**Comment by iliyan-velichkov** [comment]
official blog link - https://www.codbex.com/technology/2024/06/25/run-abap-for-free/

**Pull request merged**